WDL/CLI.py

```diff
@@ -283,12 +283,27 @@ def fill_cromwell_subparser(subparsers):
         action="append",
         help="explicitly set an array input to the empty array",
     )
+    cromwell_parser.add_argument(
+        "-r",
+        "--jar",
+        metavar="jarfile",
+        dest="jarfile",
+        type=str,
+        help="Cromwell jarfile filename/URI",
```

Comment:
1. Help message can clarify that this overrides the default behavior of downloading a hardcoded version.
2. Worth mentioning the CROMWELL_JAR env var alternative here too
3. Can this really be a URI instead of a local filename?